webextensions/api/scripting.json

```diff
@@ -165,6 +165,83 @@
               "safari_ios": "mirror"
             }
           },
+          "InjectionResult": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "102"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15.4"
+                },
```

Comment:
```suggestion
                "safari": {
                  "version_added": false,
                  "notes": "Safari returns an array with the results, instead of an array of InjectionResults."
                },
```